doc/Makefile

```diff
@@ -1,153 +1,20 @@
-# Makefile for Sphinx documentation
+# Minimal makefile for Sphinx documentation
```

Comment:
AFAIU, we redirect all existing targets to Sphinx, so this new Makefile is the sane as old one in terms of usability. So I think "minimal" addition is not required.

Reply:
It is generated Makefile, I didn't touch it. Sphinx developers highlighted that it is minimal (I guess as opposite to one generated by `sphinx-quickstart <...> --no-use-make-mode`), so I would prefer to leave it as is.

Reply:
Okay, no questions then